lib/api_local.ml

```diff
@@ -170,12 +170,15 @@ module Slack_json : Api.Slack = struct
 end
 
 module Buildkite : Api.Buildkite = struct
-  let get_job_log ~ctx:_ (job : Buildkite_t.job) =
+  let get_job_log ~ctx:_ (_ : Github_t.status_notification) (job : Buildkite_t.job) =
+    (* using the job's log_url for the local api is better than constructing it
+       because you can cheat a little to having needing 32 cached job logs files.
+    *)
```

Comment:
unneeded. It's not expected that the test function implementation reads exactly the same properties as the lib one.